app/src/main/java/com/ataulm/artcollector/EventObserver.kt

```diff
@@ -0,0 +1,22 @@
+package com.ataulm.artcollector
+
+import android.arch.lifecycle.Observer
+
+class EventObserver<T>(private val handleEvent: (T) -> Unit) : Observer<Event<T>> {
```

Comment:
similar to https://gist.github.com/JoseAlcerreca/e0bba240d9b3cffa258777f12e5c0ae9